tests/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
```

Comment:
Is there any advantage to use a crate versus a vanilla rust integration test?

Reply:
I guess I implicitly expect tests to be self contained and not require an external service to started for just one test. We could also make the test a binary then it would be more straightforward to customize it's functionality using clap.

Reply:
Yea I'm not married to the separate crate. But the test does cover more than just sequencer crate, so id does seem workspace level to me. Keep in mind that the follow up to this is upgrade tests, so there will be more in there than just one test. The idea is that there is a suit of integration tests that require native-demo to be running, this is just the first one. I don't love the idea of making this a binary. I don't think we need clap, or much customization. If we need another test, we write another test. Making it a binary will mean we either have to shoehorn new tests in to a single binary or create a new binary for every test.